README.md

```diff
@@ -202,6 +202,7 @@ Three types of these events exist:
 1. **Search Events** measure user interaction with search results
 1. **Browse Events** measure user interaction with browse results
 1. **Recommendation Events** measure user interaction with recommendations
+1. **Quiz Events** measure user interaction with quiz results
```

Comment:
I'm not sure if it was already like this, but why are all of these `1.`? Haha

It doesn't show up as `1.` for all of them in the actual [README](https://github.com/constructor-io/constructorio-client-swift#12-instrument-behavioral-events) though which is interesting. I guess it just works? 😂

Reply:
I have no idea haha I saw it too but it wasn't broken so I just let it be 🤷. I'll update it.